src/game/version.h

```diff
@@ -5,5 +5,6 @@
 #include <generated/nethash.cpp>
 #define GAME_VERSION "0.7 trunk"
 #define GAME_NETVERSION "0.7 " GAME_NETVERSION_HASH
+#define GAME_NETGUI_VERSION "0.7-0.1.b20151228"
```

Comment:
Oh, it should have been removed yet, it's a leftover from my own version checking system that was removed for ModAPI... totally forgot about it :D